src/Raven.Server/Documents/Sharding/Handlers/Processors/OngoingTasks/ShardedOngoingTasksHandlerProcessorForGetOngoingTasks.cs

```diff
@@ -15,25 +16,26 @@ public ShardedOngoingTasksHandlerProcessorForGetOngoingTasks([NotNull] ShardedDa
         {
         }
 
-        protected override bool SupportsCurrentNode => true;
+        protected override bool SupportsCurrentNode => false;
 
-        protected override async ValueTask HandleCurrentNodeAsync()
+        protected override RavenCommand<OngoingTasksResult> CreateCommandForNode(string nodeTag)
         {
-            var result = GetOngoingTasksInternal();
-
-            using (ContextPool.AllocateOperationContext(out JsonOperationContext context))
-            await using (var writer = new AsyncBlittableJsonTextWriter(context, RequestHandler.ResponseBodyStream()))
-            {
-                context.Write(writer, result.ToJson());
-            }
+            return new GetOngoingTasksInfoCommand(nodeTag);
         }
 
-        protected override RavenCommand<OngoingTasksResult> CreateCommandForNode(string nodeTag)
+        protected override bool IsCurrentNode(out string nodeTag)
         {
-            return new GetOngoingTasksInfoCommand(nodeTag);
+            nodeTag = GetNodeTag(required: false);
```

Comment:
if this is null and you return false from here then what nodeTag we will pass?

Reply:
`null` :)

Reply:
Let me rephrase myself, what node will be picked then?

Reply:
the first in the topology of the shard.
e.g. test$1 has the topology of [B,E] -> node `B` will be picked.